### ISSUE TITLE
[BUG] rt1308 initialization timed out on TGLU_RVP_SDW when suspend-resume

### ISSUE THREAD

**Issue by ghost** [thesofproject/linux#3140]
**Describe the bug**
On TGLU_RVP_SDW, rt1308 initialization timed out when system try resuming after suspend. This issue was found in the stress test for [tgl/v13.0-hotfix3](https://github.com/thesofproject/sof/tree/releases/tgl%2Fv13.0-hotfix3).

**To Reproduce**
Check dmesg log after suspend-reusme.

**Reproduction Rate**
Only once in 190 iterations of suspend-resume.

**Environment**
Kernel Branch: topic/sof-dev
Kernel Commit: 6a95ae1a
SOF Branch: tgl-013-drop-stable
SOF Commit: <https://github.com/thesofproject/sof/commit/364f715b41a5>
Topology: sof-tgl-rt711-rt1308-4ch.tplg
Platform: TGLU_RVP_SDW

**Screenshots or console output**
dmesg:

```
[37219.536761] kernel: rt1308 sdw:1:025d:1308:00:0: Initialization not complete, timed out
[37219.536767] kernel: PM: dpm_run_callback(): acpi_subsys_resume+0x0/0x70 returns -110
[37219.536778] kernel: rt1308 sdw:1:025d:1308:00:0: PM: failed to resume: error -110
[37219.541556] kernel: OOM killer enabled.
[37219.541558] kernel: Restarting tasks ... done.
[37219.546408] kernel: soundwire sdw-master-1: clock stop prep/de-prep done slave:15
[37219.546514] kernel: soundwire sdw-master-0: clock stop prep/de-prep done slave:15
[37219.546741] kernel: soundwire_intel soundwire_intel.link.0: intel_link_power_down: powering down all links
[37219.546810] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: FW Poll Status: reg[0x80]=0x20140000 successful
[37219.552093] kernel: rfkill: input handler enabled
[37219.553022] kernel: PM: suspend exit
```

[full dmesg](https://github.com/thesofproject/linux/files/7114257/full_dmesg.txt)

**Comment by plbossart** [comment]
It's a known issue, duplicate of  https://github.com/thesofproject/linux/issues/3063

````
[37213.705078] kernel: rt1308 sdw:1:025d:1308:00:2: sdw_modify_slave_status: initializing enumeration and init completion for Slave 1
[37213.705083] kernel: rt1308 sdw:1:025d:1308:00:0: sdw_modify_slave_status: initializing enumeration and init completion for Slave 2
[37213.706118] kernel: soundwire_intel soundwire_intel.link.1: Slave status change: 0x2
[37213.706123] kernel: soundwire sdw-master-1: Slave attached, programming device number
[37213.706353] kernel: soundwire sdw-master-1: SDW Slave Addr: 20025d130800
[37213.706354] kernel: soundwire sdw-master-1: SDW Slave class_id 0x00, mfg_id 0x025d, part_id 0x1308, unique_id 0x0, version 0x2
[37213.706356] kernel: soundwire sdw-master-1: Slave already registered, reusing dev_num:2
[37213.706912] kernel: soundwire_intel soundwire_intel.link.1: Msg ignored for Slave 0
[37213.706914] kernel: soundwire sdw-master-1: No more devices to enumerate
[37213.706999] kernel: rt1308 sdw:1:025d:1308:00:0: sdw_modify_slave_status: signaling enumeration completion for Slave 2
[37213.843863] kernel: rt1308 sdw:1:025d:1308:00:0: rt1308_io_init m_btl_l=0xffe8, m_btl_r=0xfffa
[37213.843869] kernel: rt1308 sdw:1:025d:1308:00:0: rt1308_io_init c_btl_l=0x3bd, c_btl_r=0x170
[37213.855164] kernel: rt1308 sdw:1:025d:1308:00:0: rt1308_io_init hw_init complete
[37213.855171] kernel: rt1308 sdw:1:025d:1308:00:0: sdw_handle_slave_status: signaling initialization completion for Slave 2
<<< Slave2 is initialized
[37213.855224] kernel: rt1308 sdw:1:025d:1308:00:0: sdw_modify_slave_status: initializing enumeration and init completion for Slave 2 
<<<<< THIS IS NOT NORMAL!
[37213.855233] kernel: soundwire sdw-master-1: Slave attached, programming device number
[37213.855550] kernel: soundwire sdw-master-1: SDW Slave Addr: 22025d130800
[37213.855552] kernel: soundwire sdw-master-1: SDW Slave class_id 0x00, mfg_id 0x025d, part_id 0x1308, unique_id 0x2, version 0x2
[37213.855555] kernel: soundwire sdw-master-1: Slave already registered, reusing dev_num:1
[37213.855936] kernel: soundwire_intel soundwire_intel.link.1: Msg ignored for Slave 0
[37213.855939] kernel: soundwire sdw-master-1: No more devices to enumerate
[37213.856017] kernel: rt1308 sdw:1:025d:1308:00:2: sdw_modify_slave_status: signaling enumeration completion for Slave 1
[37213.987853] kernel: rt1308 sdw:1:025d:1308:00:2: rt1308_io_init m_btl_l=0x0, m_btl_r=0x0
[37213.987860] kernel: rt1308 sdw:1:025d:1308:00:2: rt1308_io_init c_btl_l=0x0, c_btl_r=0x0
[37214.000043] kernel: rt1308 sdw:1:025d:1308:00:2: rt1308_io_init hw_init complete
[37214.000047] kernel: rt1308 sdw:1:025d:1308:00:2: sdw_handle_slave_status: signaling initialization completion for Slave 1
[37219.536761] kernel: rt1308 sdw:1:025d:1308:00:0: Initialization not complete, timed out
[37219.536767] kernel: PM: dpm_run_callback(): acpi_subsys_resume+0x0/0x70 returns -110
[37219.536778] kernel: rt1308 sdw:1:025d:1308:00:0: PM: failed to resume: error -110
````

**Comment by keqiaozhang** [comment]
Close this one, duplicate of #3063.
